api/openapi.yaml

```diff
@@ -500,3 +522,19 @@ components:
           example: "this is an error message"
       required:
         - message
+
+    GoodHealthCheckResponse:
+      type: object
+      additionalProperties: false
+      properties:
+        error:
```

Comment:
I added error to the good health check response to be consistent with our response other objects, I will create a task to revisit these objects